ansible/roles/operator-pipeline/tasks/operator-pipeline-event-listener.yml

```diff
@@ -59,6 +59,48 @@
                   - ref: operator-hosted-pipeline-trigger-binding
                 template:
                   ref: operator-hosted-pipeline-trigger-template
+              # run community hosted pipeline on PR opened, reopened, synchronized
+              - name: github-pull-request-listener
+                interceptors:
+                  - github:
+                      secretRef:
+                        secretName: github-webhook-secret
+                        secretKey: webhook-secret
+                      eventTypes:
+                        - pull_request
+                  - cel:
+                      filter: >-
+                        (
+                          header.match("X-GitHub-Event", "pull_request")
+                          && body.action in ["opened", "reopened", "synchronize"]
+                          && body.pull_request.base.ref == "{{ branch }}"
+                        )
+                bindings:
+                  - ref: community-operator-hosted-pipeline-trigger-binding
+                template:
+                  ref: community-operator-hosted-pipeline-trigger-template
+              # run community release pipeline on merged PR
+              - name: github-pull-request-listener
+                interceptors:
+                  - github:
+                      secretRef:
+                        secretName: github-webhook-secret
+                        secretKey: webhook-secret
+                      eventTypes:
+                        - pull_request
+                  - cel:
+                      filter: >-
+                        (
+                          header.match("X-GitHub-Event", "pull_request")
+                          && body.action == "closed"
+                          && body.pull_request.base.ref == "{{ branch }}"
+                          && body.pull_request.merged == true
+                          && body.sender.login == "rh-operator-bundle-bot"
+                        )
+                bindings:
+                  - ref: community-operator-release-pipeline-trigger-binding
+                template:
+                  ref: community-operator-release-pipeline-trigger-template
```

Comment:
Can you move this to a dedicated event listener for community pipelines? I'd rather leave the ISV stuff unchanged as much as possible at this stage.

Reply:
event listener was separated from the operator-pipelines